metadata.json

```diff
@@ -14,6 +14,13 @@
         "12.04",
         "14.04"
       ]
+    },
+    {
+      "operatingsystem": "RedHat",
+      "operatingsystemrelease": [
```

Comment:
As you can see here : https://github.com/puppetlabs/puppetlabs-ntp/blob/master/metadata.json
You can keep the 'operatingsystemrelease' keyword but you have to remove '.0' in this versions